src/plugins/algolia-init-mixin.js

```diff
@@ -14,13 +14,25 @@ const indexKeys = [
 
 // Rebuild index name as set in Algolia
 function setSortByEnv(selectedRouteSort) {
-	let expandedRouteSort = '';
-	['dev', 'qa', 'prod'].forEach(env => {
-		if (typeof window !== 'undefined' && window.location.host.indexOf(env) !== -1) {
```

Comment:
I noticed this was only happening on prod, this line was checking for prod.kiva.org as the host name